script/QA/QA_12.txt

```diff
@@ -113,7 +113,7 @@ The answers to the mysteries that remain,
 [sha:b1d91e2eabdeae6ce168a6767b9241c065fc1cd9]{
 -- Page 302, Offset 43314.
 -- 『月の表』、そのすべてを見せる<物語|ルート>で、
-Let's meet again in the next route, the%{n}'Near Side of the Moon',
+Let's meet again in the next route of the%{n}'Near Side of the Moon',
```

Comment:
the line as it was suggests that Ciel's route is the 'Near Side of the Moon' so this was just a quick hack. Not particularly fond of it.

Reply:
I think this change is fine.